slither/solc_parsing/cfg/node.py

```diff
@@ -62,9 +66,19 @@ def analyze_expressions(self, caller_context: Union["FunctionSolc", "ModifierSol
 
             find_call = FindCalls(expression)
             self._node.calls_as_expression = find_call.result()
-            self._node.external_calls_as_expressions = [
-                c for c in self._node.calls_as_expression if not isinstance(c.called, Identifier)
-            ]
-            self._node.internal_calls_as_expressions = [
-                c for c in self._node.calls_as_expression if isinstance(c.called, Identifier)
-            ]
+
+            def is_external_call(element: "Expression") -> bool:
+                if not isinstance(element.called, Identifier):
+                    try:
+                        return not isinstance(element.called.expression.value, SolidityVariable)
```

Comment:
I think this should be able to be `isinstance(element.called.value, SolidityVariable)` and not require a try/except

Reply:
That was my initial try but the test failed on the CI : 
https://github.com/crytic/slither/actions/runs/8649176961/job/23714782903